src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
 public class Application {
+    public static final int numberOfBall = 3;
```

Comment:
```suggestion
    private static final int numberOfBall = 3;
```
Use upper case for constant name